js/src/forum/index.js

```diff
@@ -9,10 +9,19 @@ import LastDiscussionWidget from './components/LastDiscussionWidget';
 import StatWidget from './components/StatWidget';
 
 function pruneIndexNav(items, func) {
-  if (app.current.matches(CategoriesPage) && app.forum.attribute('categories.fullPageDesktop')) {
+  const isTagsVisible = !app.forum.attribute('categories.keep-tags-nav');
```

Comment:
Why is this negated?

Reply:
Removing the tag navigation should not be checked if fullPageDesktop is true, hence why it's removed.  This caused the Tag navigation to be shown for the user.

Reply:
Ah, I agree about the fullPageDesktop change. But you have `isTagsVisible` being set to the opposite of whether the user wants to keep tags. Shouldn't it be:

`isTagsVisible = app.forum.attribute(... keep-tags-nav)`?

Reply:
It seems I've used the wrong attributes, the latest commit is 'categories.keepTagsNav' and fixed the logic